include/quicr/detail/ctrl_message_types.h

```diff
@@ -118,15 +118,27 @@ namespace quicr::messages {
         return buffer;
     }
 
-    enum struct ParameterType : uint64_t
+    enum struct SetupParameterType : uint64_t
     {
-        kPath = 0x1,
-        kMaxRequestId = 0x2, // version specific, unused
-        kEndpointId = 0xF1,  // Endpoint ID, using temp value for now
-        kInvalid = 0xFF,     // used internally.
+        kPath = 0x01,
+        kMaxRequestId = 0x02, // version specific, unused
+        kEndpointId = 0xF1,   // Endpoint ID, using temp value for now
+        kInvalid = 0xFF,      // used internally.
     };
 
-    using Parameter = KeyValuePair<ParameterType>;
+    enum struct VersionSpecificParameterType : uint64_t
+    {
+        kDeliveryTimeout = 0x02,
+        kAuthorizationToken = 0x03,
```

Comment:
I see that this one is kind of weird to talk about, in MoQ they say this is setup, but actually can appear basically everywhere (CLIENT_SETUP, SERVER_SETUP, SUBSCRIBE, SUBSCRIBE_NAMESPACE, ANNOUNCE, TRACK_STATUS or FETCH). Maybe we should track such a thing one in yet another enum. Or in both. moq pls.